client/aenea/alias.py

```diff
@@ -154,7 +155,7 @@ def split(self, text):
         self._update_regex()
         k = 0 # The end of the previous match.
         open_angle_brackets = 0
-
+        
```

Comment:
It seems like Alex doesn't mind, so I won't bring it up again. My editor shows whitespace, so spurious whitespace looks noisy. But, in any event, if you didn't know, you can have git diff show [colored output](http://jblevins.org/log/git-colors) so whitespace is highlighted. And you can use `git add -p` or `git checkout -p` to selectively add or remove parts of a diff so you can avoid adding stuff you didn't intend. If you already knew that, no worries -- I'm not trying to patronizing, I've just found a lot of people don't know and the git defaults aren't the best.